vault/request_forwarding.go

```diff
@@ -354,6 +356,7 @@ func (c *Core) ForwardRequest(req *http.Request) (int, http.Header, []byte, erro
 	}
 	resp, err := c.rpcForwardingClient.ForwardRequest(req.Context(), freq)
 	if err != nil {
+		metrics.IncrCounter([]string{"ha", "rpc", "client", "forward", "errors"}, 1)
```

Comment:
Are the other local errors happening before this intentionally not being added to metrics?

Reply:
Yeah, I wanted to track failures to forward, not so much failures to set up forwarding.